src/NUnitFramework/tests/Internal/Results/TestResultGeneralTests.cs

```diff
@@ -160,8 +160,8 @@ public void TestResultXml_Duration()
         {
             TNode testNode = _testResult.ToXml(true);
 
-            Assert.AreEqual(EXPECTED_START.ToString("u"), testNode.Attributes["start-time"]);
-            Assert.AreEqual(EXPECTED_END.ToString("u"), testNode.Attributes["end-time"]);
+            Assert.AreEqual(EXPECTED_START, DateTime.Parse(testNode.Attributes["start-time"], CultureInfo.InvariantCulture).ToUniversalTime());
+            Assert.AreEqual(EXPECTED_END, DateTime.Parse(testNode.Attributes["end-time"], CultureInfo.InvariantCulture).ToUniversalTime());
```

Comment:
For our tests, I think the first way was better. Wouldn't these tests pass even if you hadn't made the change in TestResult.cs?

Reply:
I returned back everything as was, just changed `u` to `o`. Tests are passing. Also didn't use `InvariantCulture` when converting DateTime to String, seems `o` formatter is already InvariantCulture.

Reply:
Oh, nice to know!